app/controllers/comments_controller.rb

```diff
@@ -1,69 +1,20 @@
 class CommentsController < ApplicationController
-  before_action :set_comment, only: %i[show edit update destroy]
-
-  # GET /comments or /comments.json
-  def index
-    @comments = Comment.all
-  end
-
-  # GET /comments/1 or /comments/1.json
-  def show; end
-
-  # GET /comments/new
-  def new
-    @comment = Comment.new
-  end
-
-  # GET /comments/1/edit
-  def edit; end
-
-  # POST /comments or /comments.json
   def create
```

Comment:
- [ ] Please add the new action as mentioned in the requirements. It should look similar to this
```
  def new
    @comment = Comment.new
  end

```

Reply:
Hi, @Lucas-Masaba , Please see it in the data_model branch, not in development branch
![image](https://user-images.githubusercontent.com/22744775/200106335-153e2d24-34d8-4594-83f6-7020646430b4.png)